mat/vector.go

```diff
@@ -595,6 +595,11 @@ func (v *VecDense) MulVec(a Matrix, b Vector) {
 		}
 		v.setVec(0, sum)
 		return
+	case *SymBandDense:
+		if fast {
```

Comment:
Don't we need to check overlap? It seems like in general it would be best to do this before the switch though.

Reply:
I've now added overlap detection for SymBandDense (@kortschak could you please take a look?) and at the moment use it here inside the switch case.

Reply:
This looks fine.